Jenkinsfile

```diff
@@ -116,17 +108,21 @@ node('ca-jenkins-agent') {
      def INTEGRATION_TEST_ROOT= "__tests__/__results__/integration"
      def INTEGRATION_JUNIT_OUTPUT = "$INTEGRATION_TEST_ROOT/junit.xml"
      // Perform a unit test and capture the results
-        pipeline.test(
-            name: "Integration",
-            operation: {
-                sh "npm i -g @zowe/cli@daily --zowe:registry=${pipeline.registryConfig[0].url}"
-                // create the custom properties file. contents don't matter for integration tests
-                sh "cp __tests__/__resources__/properties/example_properties.yaml __tests__/__resources__/properties/custom_properties.yaml"
-                sh "npm run test:integration"
-            },
-            testResults: [dir: "${INTEGRATION_TEST_ROOT}/jest-stare", files: "index.html", name: "${PRODUCT_NAME} - Integration Test Report"],
-            junitOutput: INTEGRATION_JUNIT_OUTPUT,
-        )
+    pipeline.test(
+        name: "Integration",
+        operation: {
+            sh "npm i -g @zowe/cli@daily --zowe:registry=${pipeline.registryConfig[0].url}"
```

Comment:
Is this supposed to be daily?

Reply:
I made it `latest` since we will be getting rid of daily hopefully pretty soon (by next week)

Reply:
Forget my previous comment.
It should be `latest`
Thanks for catching that. I thought we were talking about the dependencies.